libcontainer/container_linux.go

```diff
@@ -366,17 +381,15 @@ func (c *linuxContainer) start(process *Process, isInit, isCreate bool) error {
 		}
 		c.initProcessStartTime = state.InitProcessStartTime
 
-		if c.config.Hooks != nil && len(c.config.Hooks.Poststart) > 0 {
+		if c.config.Hooks != nil && len(c.config.Hooks.Prestart) > 0 {
 			s, err := c.currentOCIState()
 			if err != nil {
 				return err
 			}
-			for i, hook := range c.config.Hooks.Poststart {
+			s.Status = "created"
```

Comment:
Do we still need this clobber now that these are called in the right place and no longer need to work around #1715?

Reply:
@wking Yes, it makes sense. Will remove this line.
Thanks for pointing to #1715, which looks like another can of worms. ;-)